test/functionalTest/cases/0000_ngsild/ngsild_registration_get_with_URI_param_type.test

```diff
@@ -0,0 +1,381 @@
+      "relationships": [ "isParked" ]
+    }
+  ],
+  "endpoint": "http://my.csource.org:1026"
+}'
+orionCurl --url /ngsi-ld/v1/csourceRegistrations -X POST --payload "$payload"
+echo
+echo
+
+
+payload='{
+  "id": "urn:ngsi-ld:reg02",
+  "type": "ContextSourceRegistration",
+  "name": "reg_02",
+  "description": "description of reg 02",
+  "information": [
+    {
+      "entities": [
+        {
+          "id": "urn:ngsi-ld:Vehicle:A456",
+          "type": "T2"
+        }
+      ],
+      "properties": [ "brandName", "speed" ],
+      "relationships": [ "isParked" ]
+    }
+  ],
+  "endpoint": "http://my.csource.org:1026"
+}'
+orionCurl --url /ngsi-ld/v1/csourceRegistrations -X POST --payload "$payload"
+echo
+echo
+
+payload='{
+  "id": "urn:ngsi-ld:reg03",
+  "type": "ContextSourceRegistration",
+  "name": "reg_03",
+  "description": "description of reg 03",
+  "information": [
+    {
+      "entities": [
+        {
+          "id": "urn:ngsi-ld:Vehicle:A456",
+          "type": "T3"
+        }
+      ],
+      "properties": [ "brandName", "speed" ],
+      "relationships": [ "isParked" ]
+    }
+  ],
+  "endpoint": "http://my.csource.org:1026"
+}'
+orionCurl --url /ngsi-ld/v1/csourceRegistrations -X POST --payload "$payload"
+echo
+echo
+
+
+payload='{
+  "id": "urn:ngsi-ld:reg04",
+  "type": "ContextSourceRegistration",
+  "name": "reg_04",
+  "description": "description of reg 04",
+  "information": [
+    {
+      "entities": [
+        {
+          "id": "urn:ngsi-ld:Vehicle:A456",
+          "type": "T4"
+        }
+      ],
+      "properties": [ "brandName", "speed" ],
+      "relationships": [ "isParked" ]
+    }
+  ],
+  "endpoint": "http://my.csource.org:1026"
+}'
+orionCurl --url /ngsi-ld/v1/csourceRegistrations -X POST --payload "$payload"
+echo
+echo
+
+
+payload='{
+  "id": "urn:ngsi-ld:reg05",
+  "type": "ContextSourceRegistration",
+  "name": "reg_05",
+  "description": "description of reg 05",
+  "information": [
+    {
+      "entities": [
+        {
+          "id": "urn:ngsi-ld:Vehicle:A456",
+          "type": "T5"
+        }
+      ],
+      "properties": [ "brandName", "speed" ],
+      "relationships": [ "isParked" ]
+    }
+  ],
+  "endpoint": "http://my.csource.org:1026"
+}'
+orionCurl --url /ngsi-ld/v1/csourceRegistrations -X POST --payload "$payload"
+echo
+echo
+
+
+echo "02. mongo content - see 5 regs"
+echo "=============================="
+mongoCmd2 ftest "db.registrations.find()"
+echo
+echo
+
+
+echo "03. Get registration 02 using URI param type=T1"
+echo "==============================================="
+orionCurl --url "/ngsi-ld/v1/csourceRegistrations?type=T1&prettyPrint=yes&spaces=2" --noPayloadCheck
+echo
+echo
+
+
+echo "04. Get registration 01, 03, and 05, using URI param type=T1, T3, T5"
+echo "===================================================================="
+orionCurl --url "/ngsi-ld/v1/csourceRegistrations?type=T1,T2,T3,T5&prettyPrint=yes&spaces=2" --noPayloadCheck
```

Comment:
The comment (echo) says T1,T3,T5 but the call to orionCurl contains also T2.
The test is OK, but the comment should change - in 3 places

Reply:
Right! Thanks

Reply:
Fixed in e86972c9e5a12cb4169cbec576aea256f63e211b